NGitLab.Mock/Fluent/GitLabUser.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace NGitLab.Mock.Fluent
+{
+    public class GitLabUser : GitLabObject<GitLabConfig>
+    {
+        public string Username { get; set; }
+
+        public string Name { get; set; } = "User";
+
+        public string Email { get; set; } = "user@ubisoft.com";
```

Comment:
You should use "example.com" domain

```suggestion
        public string Email { get; set; } = "user@example.com";
```